src/ui/map.js

```diff
@@ -2319,6 +2319,7 @@ class Map extends Camera {
         }
         this._renderTaskQueue.clear();
         this.painter.destroy();
+        this.handlers.destroy();
```

Comment:
might be worth setting `this.handlers = null` to break the cycle, since handlers caches a reference to map.

Reply:
e91720c